src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Helpers/StringUrlHelperTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using NUnit.Framework;
+using Sfa.Das.Sas.Web.Helpers;
+using Assert = NUnit.Framework.Assert;
+
+namespace Sfa.Das.Sas.Web.UnitTests.Infrastructure.Web.Helpers
+{
+    [TestFixture]
+    public class StringUrlHelperTests
+    {
+        [TestCase("WOODSPEEN TRAINING LIMITED","woodspeen-training-limited")]
+        [TestCase("5 E LTD.", "5-e-ltd")]
+        [TestCase("TRN(TRAIN) LTD.", "trn-train-ltd")]
+        [TestCase("ACTIVE LEARNING & DEVELOPMENT LIMITED", "active-learning-and-development-limited")]
+        [TestCase("BARKING & DAGENHAM LONDON BOROUGH COUNCIL", "barking-and-dagenham-london-borough-council")]
+        [TestCase("BARNARDO'S", "barnardos")]
+        [TestCase("BASSETLAW TRAINING AGENCY(PROPERTIES) LIMITED", "bassetlaw-training-agency-properties-limited")]
+        [TestCase("BERKSHIRE COLLEGE OF AGRICULTURE, THE(BCA)", "berkshire-college-of-agriculture-the-bca")]
+        [TestCase("BOURNEMOUTH AND POOLE COLLEGE, THE", "bournemouth-and-poole-college-the")]
+        [TestCase("C.M.S.VOCATIONAL TRAINING LIMITED", "c-m-s-vocational-training-limited")]
+        [TestCase("EALING, HAMMERSMITH & WEST LONDON COLLEGE", "ealing-hammersmith-and-west-london-college")]
+        [TestCase("EAST MIDLANDS CHAMBER(DERBYSHIRE, NOTTINGHAMSHIRE, LEICESTERSHIRE)", "east-midlands-chamber-derbyshire-nottinghamshire-leicestershire")]
+        [TestCase("E.QUALITY TRAINING LIMITED", "e-quality-training-limited")]
+        [TestCase("E.Q.V. (UK) LIMITED", "e-q-v-uk-limited")]
+        [TestCase("FASHION - ENTER LTD", "fashion-enter-ltd")]
+        [TestCase("MARITIME + ENGINEERING COLLEGE NORTH WEST", "maritime-and-engineering-college-north-west")]
+        [TestCase(" N & B TRAINING LTD", "n-and-b-training-ltd")]
+        [TestCase("GREENBANK PROJECT(THE)", "greenbank-project-the")]
+        [TestCase("L.I.T.S.LIMITED", "l-i-t-s-limited")]
```

Comment:
for L.I.T.S it should probably resolve as lits-limited since I'd search google for "lits", sorry if this adds complexity

Reply:
I've done this, but it creates problems.  For example:
 "L.I.T.S.LIMITED" becomes 'litslimited' 
"C.M.S.VOCATIONAL TRAINING LIMITED" becomes 'cmsvocational-training-limited'
"NORTH LANCS.TRAINING GROUP LIMITED(THE)" becaomes 'north-lancstraining-group-limited-the'

I'll put the change in, and we'll discuss once other reworking done?